helix-term/src/keymap.rs

```diff
@@ -377,3 +382,233 @@ pub fn default() -> Keymaps {
         ),
     )
 }
+
+// Newtype wrapper over keys to allow toml serialization/parsing
+#[derive(Debug, PartialEq, PartialOrd, Clone, Copy, Hash)]
+pub struct RepresentableKeyEvent(pub KeyEvent);
+impl Display for RepresentableKeyEvent {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let Self(key) = self;
+        f.write_fmt(format_args!(
+            "{}{}{}",
+            if key.modifiers.contains(KeyModifiers::SHIFT) {
+                "S-"
+            } else {
+                ""
+            },
+            if key.modifiers.contains(KeyModifiers::ALT) {
+                "A-"
+            } else {
+                ""
+            },
+            if key.modifiers.contains(KeyModifiers::CONTROL) {
+                "C-"
+            } else {
+                ""
+            },
+        ))?;
+        match key.code {
+            KeyCode::Backspace => f.write_str("Bs")?,
+            KeyCode::Enter => f.write_str("Enter")?,
+            KeyCode::Left => f.write_str("Left")?,
+            KeyCode::Right => f.write_str("Right")?,
+            KeyCode::Up => f.write_str("Up")?,
+            KeyCode::Down => f.write_str("Down")?,
+            KeyCode::Home => f.write_str("Home")?,
+            KeyCode::End => f.write_str("End")?,
+            KeyCode::PageUp => f.write_str("PageUp")?,
+            KeyCode::PageDown => f.write_str("PageDown")?,
+            KeyCode::Tab => f.write_str("Tab")?,
+            KeyCode::BackTab => f.write_str("BackTab")?,
+            KeyCode::Delete => f.write_str("Del")?,
+            KeyCode::Insert => f.write_str("Insert")?,
```

Comment:
Can you please try to switch it to follow kakoune https://github.com/mawww/kakoune/blob/master/src/keys.cc#L61-L85, I think it would be easier if all is lowercase and we don't have to do it in a special case.

Reply:
Yup that sounds good, unless anyone else has any objections, I'll do this.